example/call/call.go

```diff
@@ -176,14 +195,28 @@ func (app *callApp) run() {
 }
 
 func (app *callApp) consoleIO() {
+	time.Sleep(250 * time.Millisecond)
```

Comment:
maybe a comment as to what that's for? sticks out when reviewing :)

Reply:
Done